### PR TITLE
Add more min max validation, fix spacing, update UG

### PR DESCRIPTION
**Pull request by rouvinerh**

### CODE REVIEW

File: src/main/java/constants/ErrorConstant.java [hunk 1]
```diff
@@ -74,10 +74,10 @@ public class ErrorConstant {
             + "Only input what is required! Additional characters between flags will cause errors.";
     public static final String INVALID_RUN_DISTANCE_ERROR = "Distance is a 2 decimal point positive number!";
     public static final String INVALID_RUN_TIME_ERROR = "Invalid time format. Format is either HH:MM:SS or " +
-            "MM:SS with integers.";
-    public static final String INVALID_SECOND_ERROR = "Seconds must be a positive integer between 00 and 59";
-    public static final String INVALID_HOUR_ERROR = "Hours cannot set to 00. Use MM:SS instead";
-    public static final String INVALID_MINUTE_ERROR = "Minutes must be a positive integer between 01 and 59";
+            "MM:SS with integers!";
+    public static final String INVALID_SECOND_ERROR = "Seconds must be a positive integer between 00 and 59!";
+    public static final String INVALID_HOUR_ERROR = "Hours cannot set to 00. Use MM:SS instead!";
```
Comment (JustinSoh) [inline src/main/java/constants/ErrorConstant.java:79]: Typo. Hours cannot be set to 00

**Review by JustinSoh — COMMENTED** [review]
Looks good to me

File: src/main/java/constants/ErrorConstant.java (hunk 1 above)
Comment (L5-Z) [inline src/main/java/constants/ErrorConstant.java:79]: Maybe do: "Hours is excluded if less than 1. Use MM:SS instead"

File: src/main/java/constants/ErrorConstant.java [hunk 2]
```diff
@@ -98,6 +98,15 @@ public class ErrorConstant {
     public static final String INVALID_REPS_ERROR = "Number of reps must be a positive integer!"
             + System.lineSeparator()
             + GYM_STATION_FORMAT_ERROR;
+
+    public static final String INVALID_WEIGHTS_ERROR = "The weight done for each set is seperated by commas! " +
```
Comment (L5-Z) [inline src/main/java/constants/ErrorConstant.java:102]: The weights "used" instead of "done"?

Comment (L5-Z) [reply src/main/java/constants/ErrorConstant.java:102]: "separated" instead of "seperated"?

**Review by L5-Z — COMMENTED** [review]
Looks good

**Review by L5-Z — APPROVED** [review]
Looks good!

**Pull request merged**